http://www.root-servers.org/ pointed out an IPv4 renumbering of one of
the root DNS servers http://d.root-servers.org/renumber.html

This renumbering was done on 2013-01-03.  The last renumbering done
before this was on 2007-11-01.

--- deadwood-3.2.03/src/DwSocket.h	2012-12-20 13:30:37.000000000 -0500
+++ deadwood-3.2.04/src/DwSocket.h	2013-11-19 15:00:08.000000000 -0500
@@ -1,4 +1,4 @@
-/* Copyright (c) 2007-2011 Sam Trenholme
+/* Copyright (c) 2007-2013 Sam Trenholme
  *
  * TERMS
  *
@@ -19,12 +19,12 @@
 #ifndef __DW_SOCKET_DEFINED__
 #define __DW_SOCKET_DEFINED__
 
-/* This is the root server list as of 2011/11/24; the IPv4 root servers were
- * last changed 2007/11/01 */
+/* This is the root server list as of 2013/11/19; the IPv4 root servers were
+ * last changed 2013/01/03 */
 #define ROOT_SERVERS "198.41.0.4"       /* a.root-servers.net (VeriSign) */ \
                      ",192.228.79.201"  /* b.root-servers.net (ISI) */ \
                      ",192.33.4.12"     /* c.root-servers.net (Cogent) */ \
-                     ",128.8.10.90"     /* d.root-servers.net (UMaryland) */ \
+                     ",199.7.91.13"     /* d.root-servers.net (UMaryland) */ \
                      ",192.203.230.10"  /* e.root-servers.net (NASA Ames) */ \
                      ",192.5.5.241"     /* f.root-servers.net (ISC) */ \
                      ",192.112.36.4"    /* g.root-servers.net (DOD NIC) */ \
--- deadwood-3.2.04/doc/Deadwood.ej.orig	2013-11-19 15:05:49.000000000 -0500
+++ deadwood-3.2.04/doc/Deadwood.ej	2013-11-19 15:06:56.000000000 -0500
@@ -540,7 +540,7 @@
 198.41.0.4      a.root-servers.net (VeriSign) 
 192.228.79.201  b.root-servers.net (ISI) 
 192.33.4.12     c.root-servers.net (Cogent) 
-128.8.10.90     d.root-servers.net (UMaryland) 
+199.7.91.13     d.root-servers.net (UMaryland) 
 192.203.230.10  e.root-servers.net (NASA Ames) 
 192.5.5.241     f.root-servers.net (ISC) 
 192.112.36.4    g.root-servers.net (DOD NIC) 
--- deadwood-3.2.04/doc/dwood3rc-all.orig	2013-11-19 15:08:41.000000000 -0500
+++ deadwood-3.2.04/doc/dwood3rc-all	2013-11-19 15:08:54.000000000 -0500
@@ -106,7 +106,7 @@
 # cache and that maximum_cache_elements will need to be increased to store
 # a large number of these entries.
 #root_servers = {}
-#root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 128.8.10.90,"
+#root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 199.7.91.13,"
 #root_servers["."]+="192.203.230.10, 192.5.5.241, 192.112.36.4, 128.63.2.53, "
 #root_servers["."]+="192.36.148.17, 192.58.128.30, 193.0.14.129, 199.7.83.42, "
 #root_servers["."]+="202.12.27.33"
--- deadwood-3.2.03/doc/Windows/Reference.txt	2012-12-20 13:30:37.000000000 -0500
+++ deadwood-3.2.04/doc/Windows/Reference.txt	2013-11-19 15:07:34.000000000 -0500
@@ -404,7 +404,7 @@
  198.41.0.4      a.root-servers.net (VeriSign)
  192.228.79.201  b.root-servers.net (ISI)
  192.33.4.12     c.root-servers.net (Cogent)
- 128.8.10.90     d.root-servers.net (UMaryland)
+ 199.7.91.13     d.root-servers.net (UMaryland)
  192.203.230.10  e.root-servers.net (NASA Ames)
  192.5.5.241     f.root-servers.net (ISC)
  192.112.36.4    g.root-servers.net (DOD NIC)
--- deadwood-3.2.03/doc/Windows/dwood3rc.txt	2012-02-29 02:59:53.000000000 -0500
+++ deadwood-3.2.04/doc/Windows/dwood3rc.txt	2013-11-19 15:09:56.000000000 -0500
@@ -16,7 +16,7 @@
 #root_servers = {}
 # ICANN DNS root servers (Deadwood default if both root_servers and 
 # upstream_servers are not defined)
-#root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 128.8.10.90, "
+#root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 199.7.91.13, "
 #root_servers["."]+="192.203.230.10, 192.5.5.241, 192.112.36.4, 128.63.2.53, "
 #root_servers["."]+="192.36.148.17, 192.58.128.30, 193.0.14.129, 199.7.83.42, "
 #root_servers["."]+="202.12.27.33"
--- deadwood-3.2.03/doc/Windows/Deadwood-HOWTO.txt	2011-11-01 12:25:23.000000000 -0400
+++ deadwood-3.2.04/doc/Windows/Deadwood-HOWTO.txt	2013-11-19 15:09:31.000000000 -0500
@@ -22,7 +22,7 @@
 recursive_acl = "127.0.0.1/16, 192.168.1.1/24"
 chroot_dir = "/etc/maradns"
 root_servers = {}
-root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 128.8.10.90,"
+root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 199.7.91.13,"
 root_servers["."]+="192.203.230.10, 192.5.5.241, 192.112.36.4, 128.63.2.53, "
 root_servers["."]+="192.36.148.17, 192.58.128.30, 193.0.14.129, 199.7.83.42, "
 root_servers["."]+="202.12.27.33"
--- deadwood-3.2.03/doc/Deadwood-HOWTO	2011-11-01 12:25:23.000000000 -0400
+++ deadwood-3.2.04/doc/Deadwood-HOWTO	2013-11-19 15:08:05.000000000 -0500
@@ -22,7 +22,7 @@
 recursive_acl = "127.0.0.1/16, 192.168.1.1/24"
 chroot_dir = "/etc/maradns"
 root_servers = {}
-root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 128.8.10.90,"
+root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 199.7.91.13,"
 root_servers["."]+="192.203.230.10, 192.5.5.241, 192.112.36.4, 128.63.2.53, "
 root_servers["."]+="192.36.148.17, 192.58.128.30, 193.0.14.129, 199.7.83.42, "
 root_servers["."]+="202.12.27.33"
--- deadwood-3.2.03/doc/execfile-HOWTO.html	2011-12-21 14:44:37.000000000 -0500
+++ deadwood-3.2.04/doc/execfile-HOWTO.html	2013-11-19 15:10:12.000000000 -0500
@@ -32,7 +32,7 @@
 recursive_acl = "127.0.0.0/16,192.168.42.0/24"
 chroot_dir = "/etc/deadwood"
 root_servers = {}
-root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 128.8.10.90,"
+root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 199.7.91.13,"
 root_servers["."]+="192.203.230.10, 192.5.5.241, 192.112.36.4, 128.63.2.53, "
 root_servers["."]+="192.36.148.17, 192.58.128.30, 193.0.14.129, 199.7.83.42, "
 root_servers["."]+="202.12.27.33"
@@ -77,7 +77,7 @@
 recursive_acl = "127.0.0.0/16,192.168.42.0/24"
 chroot_dir = "/etc/deadwood"
 root_servers = {}
-root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 128.8.10.90,"
+root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 199.7.91.13,"
 root_servers["."]+="192.203.230.10, 192.5.5.241, 192.112.36.4, 128.63.2.53, "
 root_servers["."]+="192.36.148.17, 192.58.128.30, 193.0.14.129, 199.7.83.42, "
 root_servers["."]+="202.12.27.33"
--- deadwood-3.2.03/doc/dwood3rc	2012-02-29 02:59:53.000000000 -0500
+++ deadwood-3.2.04/doc/dwood3rc	2013-11-19 15:08:29.000000000 -0500
@@ -30,7 +30,7 @@
 #root_servers = {}
 # ICANN DNS root servers (Deadwood default if both root_servers and
 # upstream_servers are not defined)
-#root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 128.8.10.90,"
+#root_servers["."]="198.41.0.4, 192.228.79.201, 192.33.4.12, 199.7.91.13,"
 #root_servers["."]+="192.203.230.10, 192.5.5.241, 192.112.36.4, 128.63.2.53, "
 #root_servers["."]+="192.36.148.17, 192.58.128.30, 193.0.14.129, 199.7.83.42, "
 #root_servers["."]+="202.12.27.33"
